android/app/build.gradle

```diff
@@ -93,6 +93,8 @@ dependencies {
     // The version of react-native is set by the React Native Gradle Plugin
     implementation("com.facebook.react:react-android")
 
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.0"))
```

Comment:
Why do we need this?

Reply:
It's an interesting thing. I got CI build errors (using old arch? - can't remember) regarding that different dependencies were compiled with different kotlin version. Adding this dependency resolved this issue, aligns all the Kotlin SDK libraries with the same version. You can read more [here](https://medium.com/@gabrielshanahan/a-deep-dive-into-an-initial-kotlin-build-gradle-kts-8950b81b214).